src/swagger.yml

```diff
@@ -2647,6 +2707,37 @@ paths:
               calculated_on: "2016-01-22T11:00:45"
 
 parameters:
+  AcceptHeader:
+    name: Accept
+    description: Accepted content type
+    in: header
+    type: string
+    required: true
```

Comment:
Might need to change this to false

![image](https://user-images.githubusercontent.com/7221909/47512291-2a2df780-d87c-11e8-8a47-472e57ccc8df.png)